client/src/main/java/org/mvndaemon/mvnd/client/DaemonParameters.java

```diff
@@ -491,16 +491,22 @@ private static List<CoreExtension> readCoreExtensionsDescriptor(Path multiModule
 
     private static List<CoreExtension> filterCoreExtensions(List<CoreExtension> coreExtensions) {
         Set<String> exclusions = new HashSet<>();
-        String exclusionsString =
-                systemProperty(Environment.MVND_CORE_EXTENSIONS_EXCLUDE).asString();
+        String exclusionsString = systemProperty(Environment.MVND_CORE_EXTENSIONS_EXCLUDE)
+                .asOptional()
+                .orElse(Environment.MVND_CORE_EXTENSIONS_EXCLUDE.getDefault());
         if (exclusionsString != null) {
             exclusions.addAll(Arrays.stream(exclusionsString.split(","))
                     .filter(e -> e != null && !e.trim().isEmpty())
                     .collect(Collectors.toList()));
         }
-        return coreExtensions.stream()
-                .filter(e -> !exclusions.contains(e.getGroupId() + ":" + e.getArtifactId()))
-                .collect(Collectors.toList());
+        if (!exclusions.isEmpty()) {
+            LOG.info("Excluded extensions (GA): {}", exclusions);
```

Comment:
This should definitely not be part of the output, and not 4 times.
I'll revert this chunk, or log to debug at most.